src/schema/typeDefs/user/user.graphql

```diff
@@ -1,7 +1,6 @@
 type User {
   id: ID!
   username: String!
-  password: String!
```

Comment:
use schema still needs a password but we just won't query it from FE

Reply:
Password isn't needed at all for the schema

Reply:
Ah okay. We're still allowed to store this data in database though?

Reply:
Late response, but yep still storing it in the database